src/util.cpp

```diff
@@ -73,69 +73,18 @@
 
 #include <boost/interprocess/sync/file_lock.hpp>
 #include <boost/thread.hpp>
-#include <openssl/crypto.h>
-#include <openssl/rand.h>
-#include <openssl/conf.h>
 #include <thread>
 
 // Application startup time (used for uptime calculation)
 const int64_t nStartupTime = GetTime();
 
-const char * const BITCOIN_CONF_FILENAME = "bitcoin.conf";
-const char * const BITCOIN_PID_FILENAME = "bitcoind.pid";
+const char * const BITCOIN_CONF_FILENAME = "lbrycrd.conf";
+const char * const BITCOIN_PID_FILENAME = "lbrycrd.pid";
 
 ArgsManager gArgs;
 
 CTranslationInterface translationInterface;
 
-/** Init OpenSSL library multithreading support */
```

Comment:
Why is this removed?

Reply:
OpenSSL was removed from the core code.  Every project should remove it.